include/Clipboard.h

```diff
@@ -40,6 +42,36 @@ namespace lmms::Clipboard
 		StringPair,
 		Default
 	};
+	
+	enum StringPairDataType
```

Comment:
Which one is better: `KeyType` or `PairType` or `PairDataType` or maybe `CarriedDataType`? I think `DataType` doesn't say anything about the `StringPairDrag` system and it could be confusing because there is also a `Default` mime type, but I guess this can be fixed with a comment.

I would like to get a finalized name because this change would effect 30 files.